README.rst

```diff
@@ -360,6 +360,10 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **lower_case_cast** [symfony]
```

Comment:
Indeed, we should go in consistance way.
And then, I'm not sure which way is better